packages/config/src/utils/getConfigSources.ts

```
@@ -0,0 +1,52 @@
+          enabled: true
+        } satisfies ConfigSourceOptions;
+      }
+
+      return {
+        enabled: true,
+        ...configSource,
+        priority: configSource.priority || 0,
+        name: configSource.name || camelCase(nameOf(configSource.use).replace("ConfigSource", ""))
+      } satisfies ConfigSourceOptions;
+    })
+    .sort((a, b) => a.priority! - b.priority!)
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)